.github/actions/tests/Dockerfile

```diff
@@ -7,7 +7,9 @@ LABEL \
 
 USER 0
 
-RUN dnf install -y python-devel fedora-packager gcc krb5-devel openldap-devel
+RUN dnf install -y python-devel fedora-packager gcc krb5-devel openldap-devel python3-libmodulemd \
+                   gobject-introspection-devel cairo-devel cairo-gobject-devel
```

Comment:
Why are these needed? Why not only `python3-libmodulemd`?

Reply:
This is a good question. I was confused too when running the CI. I only install python3-libmodulemd before, but met error like "....Try sudo dnf install gobject-introspection-devel " when installing pygobject

Reply:
So I am guessing it is a side-effect for installing pygobject

Reply:
Tests are running inside the tox virtualenvs (without using system site packages), so:

1. First you need `pygobject` to be installed in virtualenvs, that's why you need `pygobject` in `requirements.txt` (it's generated by pip-compile).
2. To allow pip install `pygobject` in virtualenvs, you need some `-devel` packages on system, because they're required to compile `pygobject` and its dependencies (`pycairo` and etc) while installing them within virtualenvs, thus you need to install `gobject-introspection-devel` and `cairo-gobject-devel` with yum/dnf before creating virtualenvs.
3. Fedora systems has `libmodulemd` installed by default, so once you have `pygobject` installed in virtualenvs, it's good enough to import modulemd modules within the virtualenvs. Thus you don't need to install `python3-libmodulemd` with dnf/yum.